JitsiMeetJS.js

```diff
@@ -301,9 +303,12 @@ export default _mergeNamespaceAndModule({
     createLocalTracks(options = {}, oldfirePermissionPromptIsShownEvent) {
         let promiseFulfilled = false;
 
-        const { firePermissionPromptIsShownEvent, fireSlowPromiseEvent, ...restOptions } = options;
+        const { firePermissionPromptIsShownEvent, fireSlowPromiseEvent, desktopSharingAudio, ...restOptions } = options;
         const firePermissionPrompt = firePermissionPromptIsShownEvent || oldfirePermissionPromptIsShownEvent;
 
+        if (typeof desktopSharingAudio !== undefined) {
```

Comment:
Why is this necessary here? IIRC we initialize it in RTC.js already.

Reply:
I saw that in RTCUtils.js we call screenObtainer.init(options); -> but looks like we pass all options at the start -> then we want to change an option. This is the reason that I call again screenObtainer.init(options); with the new desktopSharingAudio option. @jallamsetty1 What do you think about this? Would you have another approach to fix this?

Reply:
But doesn't it mean that once someone sets their screenshare as their virtual background, this audio share option won't be available for regular screensharing and audio-only share ?

Reply:
No. Already test that. For example, I tried to apply the option just on the virtual background. Worked. After that, I wanted to test this case that you specified and I clicked the screen share option from the bottom toolbar to see the behaviour on browser tab sharing. The audio share checkbox was there.

Reply:
I think we should find a better way or workaround for this change in Chrome's behavior where the audio share gets checked by default whenever audio is requested as this affects regular screenshare as well.